storage/src/main/java/org/apache/uniffle/storage/handler/impl/LocalFileDeleteHandler.java

```diff
@@ -33,7 +32,7 @@ public class LocalFileDeleteHandler implements ShuffleDeleteHandler {
   @Override
   public void delete(String[] storageBasePaths, String appId, String user) {
     for (String basePath : storageBasePaths) {
-      String shufflePath = ShuffleStorageUtils.getFullShuffleDataFolder(basePath, appId);
+      final String shufflePath = basePath;
```

Comment:
Why do we modify  the name of `storageBasePaths`?

Reply:
It's OK. Rename to `shuffleDataStoredPath` ?

Reply:
Ok

Reply:
Done